### ISSUE TITLE
6.6.0 fails trying to filter ipwrap

### ISSUE THREAD

**Issue by ianw** [ansible/ansible-lint#2462]
```
$ .tox/linters/bin/ansible-lint --version
WARNING: PATH altered to include /home/iwienand/programs/zuul/zuul-jobs/.tox/linters/bin
ansible-lint 6.6.0 using ansible 2.10.17
```

ansible-lint 6.6.0 is failing on tasks with ipwrap from zuul-jobs

```
roles/pull-from-intermediate-registry/tasks/main.yaml:11: jinja: The ipwrap filter requires python's netaddr be installed on the ansible controller (jinja[invalid])
```

```
roles/pull-from-intermediate-registry/tasks/main.yaml:66: jinja: The ipwrap filter requires python's netaddr be installed on the ansible controller (jinja[invalid])
```

This is a shell task that uses ipwrap

```yaml
- name: Start socat to work around https://github.com/moby/moby/issues/39033
  shell: |
    set -o pipefail
    setsid socat -d -d TCP-LISTEN:0,fork TCP:{{ buildset_registry.host | ipwrap }}:{{ buildset_registry.port }} 2>&1 | grep --line-buffered 'listening on' >  {{ zuul.executor.work_root }}/socat_port &
  args:
    executable: /bin/bash
```

(and a few others I haven't pulled out). 

I feel like this is quite likely to be a duplicate of https://github.com/ansible/ansible-lint/issues/2459 -- but this isn't a ``lookup`` plugin as such.  Filing this just in case it is a different path.
